### ISSUE TITLE
Can't make the plugin work on my Phonegap Project :'(

### ISSUE THREAD

**Issue by Nicotz06** [mapsplugin/cordova-plugin-googlemaps#1378]
Hello there!

I'm having several issues when trying to use this plugin. I googled my issues since yesterday and I didn't find any solution for my problem.

I followed this [tutorial](https://github.com/mapsplugin/cordova-plugin-googlemaps/wiki/Tutorial-for-Windows) and I use Atom IDE for my phonegap projects.

I created an android API key, did the step 5, but not the thing mentioned for visual studio (since I use Atom), copy pasted the index.html sample, and I have those console error from my browser :

![error](https://cloud.githubusercontent.com/assets/26839949/24579814/129971cc-16fd-11e7-8395-e696087594e2.png)

I may not be good at googling my problems, but sincerely, it would be very great if someone could explain me what I did wrong (or what I forgot to do!).

Sorry for my english btw!

**Comment by wf9a5m75** [comment]
This plugin works only on Android or iOS

**Comment by Nicotz06** [comment]
Mmmmmmmmmmmmmh... seems legit.

The thing is that I tryed so many things with Phonegap for google map to work that I forgot the basics. And besides, when I built an android debug APK, the map never showed!

But I tryed today and it works... The mysteries of programing...

Thank you for this reminder,, but I understand that I will not be able to debug my app until I build an apk and directly test it on the phone ?
